pkg/condition/errors.go

```diff
@@ -1,16 +1,41 @@
 package condition
 
-import "fmt"
+import (
+	"fmt"
+)
 
 type CompilationError struct {
-	Expression string
-	Cause      error
+	Cause error
 }
 
 func (e *CompilationError) Error() string {
-	return fmt.Sprintf("failed to compile condition expression '%s'", e.Expression)
+	return fmt.Sprintf("failed to compile condition expression: %v", e.Cause)
```

Comment:
```suggestion
	return fmt.Sprintf("failed to compile condition expression %v: %v", e.Expression, e.Cause)
```

Reply:
Ah never mind, it shows up

```
{
    "code": "invalid_authorization_model",
    "message": "failed to compile condition expression: ERROR: complex:2:7: Syntax error: token recognition error at: '=\n'\n |     d =\n | ......^"
}
```

Reply:
I'll tweak these errors so that we can reference the exact condition name it applies to. I think that'll be useful. I'll keep the expression out of the error message for now. That first screenshot is an error that originates from CEL which is pretty rich.